src/liftwing_api/models/liftwing_model.py

```diff
@@ -0,0 +1,21 @@
+from typing import Dict
+from abc import abstractmethod
+
+class LiftwingModel:
```

Comment:
Let's also make this an abstract class by adding this to the top:
```
from abc import ABC
```

and then the class declaration would become 
```
class LiftwingModel(ABC):
```

Reply:
made the change